### PR TITLE
Add ability to load AffirmationBlock via GraphQL.

### PR DESCRIPTION
**Pull request by DFurnes**
### What does this PR do?

This pull request adds the ability to load `AffirmationBlock` via GraphQL. Here's the "normal" version, still provided by the PHP Content API, and the new version on `blocks/:id`:

Screen Shot 2019-11-22 at 1 56 44 PM

Screen Shot 2019-11-22 at 1 56 41 PM

### Any background context you want to provide?

I have not added the ability to specify an `onClose` yet, since I want to think if there's a better way we can do that without having to pass a "single-use" prop that doesn't apply to more components. Let me know if you have any ideas!

### What are the relevant tickets/cards?

Refs [Pivotal ID #169216379](https://www.pivotaltracker.com/story/show/169216379).

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

File: resources/assets/components/Affirmation/Affirmation.js [hunk 1]
```diff
@@ -42,65 +59,67 @@ const Affirmation = ({
...
-            >
-              <h4>1 Sign-Up</h4>
-              <p>
-                Congratulations! You signed up for your first campaign...and
-                earned your first badge. NICE. Ready to earn *another* badge?
-                Complete this campaign!
-              </p>
-            </Badge>
-          ) : null}
+    {userId ? (
```
Comment (katiecrane) [inline resources/assets/components/Affirmation/Affirmation.js]: Good call!

File: resources/assets/components/ContentfulEntry/ContentfulEntry.js [hunk 2]
```diff
@@ -81,10 +81,20 @@ class ContentfulEntry extends React.Component<Props, State> {
...
         );
 
+      // @TODO: Need to figure out 'onClose'...
+      case 'AffirmationBlock':
+        return (
+          <AffirmationContainer
+            userId={window.AUTH.id}
+            {...withoutNulls(json)}
+          />
+        );
```
Comment (katiecrane) [inline resources/assets/components/ContentfulEntry/ContentfulEntry.js]: Am I reading this right - this part is not in use yet?

Comment (DFurnes) [reply resources/assets/components/ContentfulEntry/ContentfulEntry.js]: Yep, right now this is only used to display the affirmation on the `blocks/:id` route.

**Comment by DFurnes** [comment]
Thinking about this some more –– I'm not sure that the affirmation really should be a block at all, since we don't ever embed it within content, like other blocks. I think I'm going to go back and add an `affirmation` query to GraphQL so we can get at this directly.

**Comment by DFurnes** [comment]
Oh, except sometimes we do allow multiple block types to be used as "affirmations", like the `affirmationBlock` reference field on the Share Block. Hmm...

**Pull request merged**